README.md

```diff
@@ -48,19 +50,41 @@ Options:
   -?, -h, --help  Show help and usage information
 
 Commands:
-  generate-script  Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally 
-                   modify the script if desired before running it.
-                   Note: Expects GH_SOURCE_PAT or GH_PAT env variable to be set.
-  migrate-repo     Invokes the GitHub API's to migrate the repo and all PR data.
-                   Note: Expects GH_PAT and GH_SOURCE_PAT env variables to be set. GH_SOURCE_PAT is optional, if not set GH_PAT will be used 
-                   instead.
+  generate-script       Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally modify the script if desired before running it.
+  grant-migrator-role   Allows an organization admin to grant a USER or TEAM the migrator role for a single GitHub organization. The migrator role allows the role assignee to perform migrations into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
+  migrate-repo          Invokes the GitHub APIs to migrate the repo and all repo data.
+  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
+```
+
+To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
+
+## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+
+General usage will use the `generate-script` command to create a script that can be used to migrate all repositories from a GHES organization.
+
+Due to many GHES instances containing firewall rules restricting the direct API access that GEI needs to perform a migration, the GEI CLI uploads the migration's archive data to Azure Blob Storage as a step prior to starting the migration using GEI. This allows GEI to perform a migration without directly accessing the GHES instance. Because of this extra step, you'll pass in a connection string to an Azure Storage account, specified in the parameters below.
```

Comment:
```suggestion
Due to many GHES instances containing firewall rules restricting the direct API access that GEI needs to perform a migration, the GEI CLI uploads the migration's archive data to Azure Blob Storage as a step prior to starting the migration using GEI. This allows GEI to perform a migration without directly accessing the GHES instance. To get started you'll need to setup an [Azure Blob storage container](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal). Then you'll pass in a [connection string](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/api/connection-strings/storage-connection-strings) to that Azure Storage account, specified in the parameters below.
```

Reply:
Updated in https://github.com/github/gh-gei/pull/249/commits/54d28940de6d2dcc19b43775774e76fa07441cbc